conf/clusters/prod.json

```diff
@@ -29,6 +29,7 @@
       },
       "rule_processor": {
         "current_version": "$LATEST",
+        "enable_metrics": true,
```

Comment:
In doc (`docs/source/athena-deploy.rst`), it says default value is `false`.  So which one if outdate? 😄

Reply:
Docs definitely - I do not have time to add those to this PR

Reply:
Also that's related to the athena processor not the rule processor or alert processor